### ISSUE TITLE
mAP is 

### ISSUE THREAD

**Issue by yumna4** [experiencor/keras-yolo2#428]
Hi, 

I have a validation loss as low as 2.6, but mAP is 0. Why is that?

**Comment by sarthakgupta1502** [comment]
I am facing the same issue.

**Comment by FarahSaeed** [comment]
Hi, 
Were you able to solve this. I am also having same issue.
